docs/install.md

```diff
@@ -10,30 +10,44 @@ dependencies:
 2. Choose adapter based on your HTTP client. **pubspec.yaml** file:
 
 ### Dio
+
 ```yaml
 dependencies:
   alice_dio: ^1.0.4
 ```
 
 ### Chopper
+
 ```yaml
 dependencies:
   alice_chopper: ^1.0.5
 ```
 
 ### Http
+
 ```yaml
 dependencies:
   alice_http: ^1.0.4
 ```
 
 ### Http Client
+
 ```yaml
 dependencies:
   alice_http_client: ^1.0.4
 ```
 
-3. Run `get` command:
+3. Choose optional database:
+
+### Objectbox
```

Comment:
I would maybe also add a line here about the pre-requisites for ObjectBox which are all laid out in the package's readme.

Reply:
Okay I'll add objectbox here.